profiles/base.yaml

```diff
@@ -1857,6 +1857,43 @@ almanac_no_use_scripts:
 - mech-lore
 - corn-maze
 - burgle
+- safe-room
```

Comment:
Was it intentional to add `safe-room` to `almanac_no_use_scripts` in a PR about tessera?

Should `safe-room` also or instead be in the list for tessera?

Reply:
`  def retreat(ignored_npcs = [])
    return if (DRRoom.npcs - ignored_npcs).empty?

    escape_messages = [
      /You are already as far away as you can get/,
      /You retreat from combat/,
      /You sneak back out of combat/,
      /Retreat to where/,
      /There's no place to retreat to/
    ]

    retreat_messages = [
      /retreat/,
      /sneak/,
      /grip on you/,
      /grip remains solid/,
      /You try to back/,
      /You must stand first/,
      /You stop advancing/,
      /You are already/
    ]

    loop do
      case DRC.bput("retreat", *escape_messages, *retreat_messages)
      when *escape_messages
        return true
      else
        DRC.fix_standing
      end
    end
  end`

So for me the fix standing part is the issue, because people lie down for healing. So this stands you up, then safe-room puts you down again and that cycle repeats and if there is a lot of people you are going to waiting a lot longer to be healed.

So needless to say since safe-room is inside, you aren't accumulating starlight which might make your tessera ask fail anyways, its probably better to just leave safe-room in there :)

I could put other logic around it to get it work, but this is the easiest way I can think of to handle it :)

Reply:
whoops, I see what you were saying, safe-room was accidently added to the almanac section, removed that.